subworkflows/local/yaml_input.nf

```diff
@@ -103,6 +104,7 @@ workflow YAML_INPUT {
     reads_type                       = group.reads_type
     pacbio_barcodes                  = ch_barcodes
     pacbio_multiplex_codes           = group.pacbio_multiplexing_barcode_names
+    platform                         = group.platform
```

Comment:
Not in the yaml

Reply:
this is the old one, we have agreed change it to reads_type.